### PR TITLE
qallme importer

### PR DESCRIPTION
**Pull request by dag7dev**
Cumulative update:
- updated requirements
- updated lib

New insertions:
- created qallme importer

### CODE REVIEW

File: MITADS/utils/sanitize.py [hunk 1]
```diff
@@ -127,5 +127,7 @@ def clean_single_line(self, value):
           
       if(value.isupper()):
           value = value.lower()
-    
-      return value
\ No newline at end of file
+
+      value = value.lstrip().rstrip() # clean line from whitespace at the beginning / at the end
```
Comment (Mte90) [inline MITADS/utils/sanitize.py:131]: use strip that is the same thing but with less commands

File: MITADS/qallme_importer.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+print("Qallme Importer")
+print("===============")
+# downloading resource
+download_link = 'http://qallme.fbk.eu/archive/QB_IT_V1.0_TranscriptionsReferences.zip'
+
+downloader = download.Download()
+downloader = downloader.if_not_exist(download_link)
+
+# extracting files
+downloader.zip_decompress(parsedir)
```
Comment (Mte90) [inline MITADS/qallme_importer.py:40]: https://github.com/MozillaItalia/DeepSpeech-Italian-Model/blob/master/MITADS/utils/download.py#L59 it is missing the folder where to extract

Comment (Mte90) [reply MITADS/qallme_importer.py:40]: Traceback (most recent call last):
  File "qallme_importer.py", line 43, in <module>
    os.chdir(parsedir + "QB_IT_V1.0_Translations")  # name of the folder inside
FileNotFoundError: [Errno 2] No such file or directory: 'parsing/QB_IT_V1.0_Translations'

File: MITADS/qallme_importer.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+parsedir = "parsing" + os.path.sep
+
+# managing output pathname + output filename
+outdir = "output" + os.path.sep
+filename = "qallme.txt"
+
+output_file = open(outdir + filename, "w", encoding='utf-8')
+
+# printing a message
+print("\n")
```
Comment (Mte90) [inline MITADS/qallme_importer.py:30]: remove it

File: MITADS/qallme_importer.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+for s in sentences:
+    sentences[i] = sentences[i].text
+    i += 1
+
+# sanitizing line by using libs
+for line in sentences:    
+    if line is not None:    # if we are not treating an empty line
+        line = sanitizer.maybe_normalize(line, mapping_normalization)
+
+print("OK!")
```
Comment (Mte90) [inline MITADS/qallme_importer.py:67]: this can be removed

File: MITADS/qallme_importer.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+print("Now writing to " + outdir + filename + "... ", end="")
+
+# writing to output file
+for line in sentences:
+    if line is not None:
+        output_file.write(line)
+        output_file.write("\n")
+
+print("OK!\n")
```
Comment (Mte90) [inline MITADS/qallme_importer.py:78]: also this one

File: MITADS/qallme_importer.py (hunk 2 above)
Comment (dag7dev) [reply MITADS/qallme_importer.py:39]: There is a "kinda-of" bug in download.py
That file has been updated.

**Pull request merged**